packages/main/src/components/top-navigation-bar/TopNavigationBar.jsx

```diff
@@ -20,6 +21,12 @@ import nl from "./assets/language/nl.png";
 import pt from "./assets/language/pt.png";
 import world from "./assets/language/world.png";
 
+import {
+  NovuProvider,
+  PopoverNotificationCenter,
+  NotificationBell,
+  IMessage
+} from "@novu/notification-center";
```

Comment:
i believe there already a notification manager in the app

Reply:
@billmal071 help please

Reply:
Yes, there should be, if there isn't I already told someone to install one. [toast](https://www.npmjs.com/package/react-toastify). You can install this if one isn't installed.

Reply:
Oh, from the video, I think what he's trying to do is different,